:farmer:  CycloneDDS builtin_interfaces package failed with stacktrace for rosidl_dynamic_typesupport

Failing  on nightlies since build #407. 

- [409](https://build.ros2.org/view/Rci/job/Rci__nightly-cyclonedds_ubuntu_jammy_amd64/409/)
- [408](https://build.ros2.org/view/Rci/job/Rci__nightly-cyclonedds_ubuntu_jammy_amd64/408/)
- [407](https://build.ros2.org/view/Rci/job/Rci__nightly-cyclonedds_ubuntu_jammy_amd64/407/)
  </details>

**LOG:** 
  <details>

  ```
05:29:23 --- stderr: builtin_interfaces
05:29:23 CMake Error at /tmp/ws/install_isolated/rosidl_dynamic_typesupport/share/rosidl_dynamic_typesupport/cmake/ament_cmake_export_dependencies-extras.cmake:21 (find_package):
05:29:23   By not providing "Findament_cmake_ros.cmake" in CMAKE_MODULE_PATH this
05:29:23   project has asked CMake to find a package configuration file provided by
05:29:23   "ament_cmake_ros", but CMake did not find one.
05:29:23 
05:29:23   Could not find a package configuration file provided by "ament_cmake_ros"
05:29:23   with any of the following names:
05:29:23 
05:29:23     ament_cmake_rosConfig.cmake
05:29:23     ament_cmake_ros-config.cmake
05:29:23 
05:29:23   Add the installation prefix of "ament_cmake_ros" to CMAKE_PREFIX_PATH or
05:29:23   set "ament_cmake_ros_DIR" to a directory containing one of the above files.
05:29:23   If "ament_cmake_ros" provides a separate development package or SDK, be
05:29:23   sure it has been installed.
05:29:23 Call Stack (most recent call first):
05:29:23   /tmp/ws/install_isolated/rosidl_dynamic_typesupport/share/rosidl_dynamic_typesupport/cmake/rosidl_dynamic_typesupportConfig.cmake:41 (include)
05:29:23   /tmp/ws/install_isolated/rmw/share/rmw/cmake/ament_cmake_export_dependencies-extras.cmake:21 (find_package)
05:29:23   /tmp/ws/install_isolated/rmw/share/rmw/cmake/rmwConfig.cmake:41 (include)
05:29:23   /tmp/ws/install_isolated/rosidl_generator_py/share/rosidl_generator_py/cmake/ament_cmake_export_dependencies-extras.cmake:21 (find_package)
05:29:23   /tmp/ws/install_isolated/rosidl_generator_py/share/rosidl_generator_py/cmake/rosidl_generator_pyConfig.cmake:41 (include)
05:29:23   /tmp/ws/install_isolated/rosidl_core_generators/share/rosidl_core_generators/cmake/rosidl_core_generators-extras.cmake:21 (find_package)
05:29:23   /tmp/ws/install_isolated/rosidl_core_generators/share/rosidl_core_generators/cmake/rosidl_core_generatorsConfig.cmake:41 (include)
05:29:23   CMakeLists.txt:17 (find_package)
  ```

  </details>

# Details
- Operating System: Ubuntu Jammy
- DDS implementation: CycloneDDS
- Client library (if applicable):
  - builtin_interfaces

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -22,7 +22,6 @@ set(CMAKE_VERBOSE_MAKEFILE ON)
 
 
 # DEPS =============================================================================================
-find_package(ament_cmake REQUIRED)
 find_package(ament_cmake_ros REQUIRED)
 find_package(rcutils REQUIRED)
 find_package(rosidl_runtime_c REQUIRED)
@@ -65,8 +64,7 @@ install(
 )
 
 ament_export_targets(${PROJECT_NAME}-export HAS_LIBRARY_TARGET)
-ament_export_dependencies(ament_cmake)
-ament_export_dependencies(ament_cmake_ros)
+ament_export_dependencies(rcutils)
 ament_export_dependencies(rosidl_runtime_c)
 
 ament_package()

